omnibus/files/private-chef-cookbooks/private-chef/templates/default/nginx/scripts/check_token_using_endpoint.lua.erb

```diff
@@ -0,0 +1,24 @@
+ngx.req.read_body()
+local res
+
+if ngx.var.request_method == "POST" then
```

Comment:
Currently, this is not generic anyways, so do we need GET or is data-collector only accepting POSTs anyways?

I'm just saying that we might want to go either way:

1. making this truely generic (e.g. adding HEAD, OPTION, DELETE)
2. baking in data-collector specific assumptions

I'd lean towards (2.) now and (1.) later. (Creating something, maybe a proper lua module, that is tested properly and used in various places might make much sense.)